### ISSUE TITLE
CMI v8.6.8.3 generated an exception // NPE // com.Zrips.CMI.Modules.BungeeCord.BungeeCordAPI.getPlayerList

### ISSUE THREAD

**Issue by x1p** [Zrips/CMI#2129]
**Description of issue:**
This error spams all my servers since 8.6.8.3 when a player leaves/disconnects.
Points to something with bungee. Although bungee functionalities are all disabled.

---

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
[17:52:33 WARN]: [CMI] Task #77897 for CMI v8.6.8.3 generated an exception
java.lang.NullPointerException: null
at com.Zrips.CMI.Modules.BungeeCord.BungeeCordAPI.getPlayerList(BungeeCordAPI.java:137) ~[?:?]
at com.Zrips.CMI.Modules.BungeeCord.BungeeCordManager$3.run(BungeeCordManager.java:354) ~[?:?]
at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.13.2.jar:git-Paper-641]
at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:449) ~[patched_1.13.2.jar:git-Paper-641]
at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1010) ~[patched_1.13.2.jar:git-Paper-641]
at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-641]
at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940) ~[patched_1.13.2.jar:git-Paper-641]
at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837) ~[patched_1.13.2.jar:git-Paper-641]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```

**CONFIG SECTION (DELETE IF NOT RELEVANT):**
```
Chat:
  BungeeMessages: false
  BungeePublicMessages: false
  BungeeStaffMessages: false
BungeeCord:
  ServerList: []
```

---

**Cmi Version (using`/cmi version`):** 
8.6.8.3
**Server Type (Spigot/Paper/etc):** 
Paper
**Server Version (using `/ver`):** 
1.13.2 build 641

**Comment by x1p** [comment]
Seems that I'm not the only one: #2134 and #1951 
Hope this gets fixed quickly. It's pretty annoying

**Comment by 4c5458** [comment]
Agree

**Comment by antiburen** [comment]
I'm also have this error, please solve it

**Comment by x1p** [comment]
Since nobody moderates this github I might as well do it myself. Solved in 8.6.9.0 and beyond.

It only would have been nice if the owner would let us know that he had seen this and worked on a solution.
